types/artifacts.d.ts

```diff
@@ -213,6 +215,24 @@ declare global {
         requestId: string | null
       }
 
+      export type RawSourceMap = {
+        version: number
+        sources: string[]
+        names: string[]
+        sourceRoot?: string
+        sourcesContent?: string[]
+        mappings: string
+        file: string
+      }
+
+      export type SourceMap = {
```

Comment:
doc strings for these two cases would be good too

Reply:
yeah, for these fields as well, so you get nice intellisense hover working for them.